### PR TITLE
creating difficulty system

### PR DESCRIPTION
**Pull request by VictorVictini**
- MiniMax AI is incomplete
- refactored HasWon function
- need to create UI for choosing difficulty

### CODE REVIEW

**Comment by Deadly-Pigg** [comment]
yeah, I wonder why

**Comment by VictorVictini** [comment]
bruh

**Comment by Deadly-Pigg** [comment]
> bruh


Uploading of course, this is the new weapon you ordered.mp4…

**Comment by VictorVictini** [comment]
that's a long file name

File: src/Program.cs [hunk 1]
```diff
@@ -9,95 +9,149 @@ public class Program {
...
+                        break;
+                    case "hard":
+                        difficulty = AIChance.Hard;
+                        break;
+                    case "impossible":
+                        difficulty = AIChance.Impossible;
+                        break;
+                    default:
+                        Console.WriteLine("Invalid input provided. Please enter \"easy\", \"medium\", \"hard\", or \"impossible\"");
+                        reset = true;
```
Comment (mohfez) [inline src/Program.cs]: you can remove the bool and replace this part with `continue`:

```csharp
default:
    Console.WriteLine("Invalid input provided. Please enter \"easy\", \"medium\", \"hard\", or \"impossible\"");
    continue;
```

**Comment by Deadly-Pigg** [comment]
it's the new weapon you ordered

File: src/Program.cs (hunk 1 above)
Comment (VictorVictini) [reply src/Program.cs]: good point

Comment (Deadly-Pigg) [reply src/Program.cs]: pee blue

Comment (VictorVictini) [reply src/Program.cs]: added the change

**Comment by VictorVictini** [comment]
thanks boss

**Pull request merged**